Source/ResponseSerialization.swift

```diff
@@ -515,12 +515,23 @@ extension DownloadRequest {
 }
 
 // MARK: - Empty
+/// A protocol for a type representing an empty response. Use `T.emptyValue` to get an instance.
+public protocol EmptyResponse {
+    static func emptyResponseValue() -> Self
+}
 
 /// A type representing an empty response. Use `Empty.value` to get the instance.
 public struct Empty: Decodable {
     public static let value = Empty()
 }
 
+extension Empty: EmptyResponse {
+    public static func emptyResponseValue() -> Empty {
+        return value
+    }
+    public static var emptyResponseValueVar: Empty { return value }
```

Comment:
This looks to be unused.

Reply:
addressed in 4664bb578a70c3f8a1d3f74082b352d5eea8fd06